canonical-web-and-design/maas-ui
# Fix bond member mac addresses.

## Done
- Fixed bond member MAC addresses in machine details network tab

## QA
- Go to the machine details > network tab of a New machine.
- Add a couple of interfaces on the same fabric/VLAN, select them then click "Create bond".
- Under "Advanced options" change the MAC address to something different and save.
- Check that the MAC addresses under each interface name is correct.

## Fixes
Fixes #1167 

## Launchpad Issue
[lp#1870117](https://bugs.launchpad.net/maas/+bug/1870117)
